pylint/testutils/_primer/primer.py

```diff
@@ -83,219 +71,20 @@ def __init__(self, primer_directory: Path, json_path: Path) -> None:
             required=True,
             help="Commit hash of the PR commit being checked.",
         )
```

Comment:
Please keep those white lines, they really help with readability. We do the same in `astroid` for setting attributes with docstrings.